### ISSUE TITLE
Default login credentials?

### ISSUE THREAD

**Issue by Cronocide** [hcengineering/huly-selfhost#6]
What are the default admin credentials, or how are they set up the first time? I've got the accounts service up, but everything I've tried is returning `platform:status:AccountNotFound`.

**Comment by kokoali-bima** [comment]
> What are the default admin credentials, or how are they set up the first time? I've got the accounts service up, but everything I've tried is returning `platform:status:AccountNotFound`.

did you get what is the default credentials? I have no idea, where's to create it on docker container?

**Comment by aonnikov** [comment]
We have neither default credentials nor admin user created 😄 
For the first time you can create a user via UI or via command line:
```
rushx run-local create-account admin -p password -f John -l Doe
rushx run-local assign-workspace admin my-workspace
rushx run-local set-user-role admin my-workspace OWNER
```

**Comment by kokoali-bima** [comment]
I have tried to use that command as written on docs. on which container I should use that command on huly docker self host version?
I think for docker selfhosted version there is a missing container. maybe a container that we can use rushx command?

**Comment by aonnikov** [comment]
Ah, the tools are available in the sources only.

There is much simpler way to do this. Just create a new account from UI, when you create a new workspace, it will this account will be an admin account for this workspace.

**Comment by avl93** [comment]
Aren't there any "server admin" account, which could, like, manage other users, i.e. enable/disable creation of new accounts?

**Comment by aonnikov** [comment]
@avl93 nope. The user who created a workspace becomes the owner of this workspace and can manage existing accounts. We do not provide a way to disable creation of new accounts, though.

**Comment by ronzyfonzy** [comment]
I deployed this today on a ubuntu machine and got the same problem:

- don't have any default user/pass to get in
- don't have an option to create a user

Altho as explained I could create a new user via the web UI but I get the error as mentioned by @kokoali-bima reported in #15

**Comment by DrMxrcy** [comment]
> @avl93 nope. The user who created a workspace becomes the owner of this workspace and can manage existing accounts. We do not provide a way to disable creation of new accounts, though.

I agree with @Hawolex Without support to disable account creation, it causes a security nightmare.

**Comment by aonnikov** [comment]
@Hawolex @DrMxrcy we are currently working on this, there will be an announcement soon.

**Comment by lexiv0re** [comment]
Please check https://github.com/hcengineering/huly-selfhost/issues/55 on disabling sign-up.
